go.mod

```diff
@@ -1,7 +1,8 @@
 module github.com/pions/sctp
 
 require (
-	github.com/pions/transport v0.2.0
+	github.com/davecgh/go-spew v1.1.1 // indirect
```

Comment:
Maybe try removing this line followed by `go mod tidy`. I've noticed these sneaking in but I don't think they are really necessary.

Reply:
Thank you! I will do it once pions/transport#9 (this PR depends on) has merged into master.